packages/react/src/themes/teams-dark/components/Segment/segmentVariables.ts

```diff
@@ -0,0 +1,6 @@
+import { SegmentVariables } from '../../../teams/components/Segment/segmentVariables'
+
+export default (siteVars: any): Partial<SegmentVariables> => ({
+  colorDisabled: siteVars.colors.grey[450],
+  backgroundColorDisabled: siteVars.colors.grey[550],
```

Comment:
Do we have these colors in `colorScheme`?

Reply:
yes